lib/rules/key-spacing.js

```diff
@@ -141,6 +187,32 @@ module.exports = {
                 {
                     type: "object",
                     properties: {
+                        align: {
```

Comment:
Shouldn't this be part of the `align` in the options above? This would allow `{ "align": "colon", "beforeColon": true }` (above) and `{ "align": { "on": "colon", "afterColon": false } }` (this) but not `{ "align": { "on": "colon", "afterColon": false }, "beforeColon": true }`.